libs/community/langchain_community/tools/audio/base.py

```diff
@@ -0,0 +1,20 @@
+from abc import abstractmethod
+from typing import Optional
+
+from langchain_core.callbacks import CallbackManagerForToolRun
+from langchain_core.tools import BaseTool
+
+
+class AudioTool(BaseTool):
```

Comment:
we should wait to add any abstractions till we have a few more examples of what audio tools look like

Reply:
Got it - removed here https://github.com/langchain-ai/langchain/pull/18880/commits/352902bcaa452cb21d1ae8dbd7c023d988929fe0.